zip-liberated-payments.rst

```diff
@@ -0,0 +1,410 @@
+::
+
+  ZIP: XXXX
+  Title: URI-Encapsulated Payments
+  Owners: Ian Miers <imichaelmiers@gmail.com>
+          Eran Tromer <eran@tromer.org>
+  Credits: Sean Bowe
+           Deirdre Connolly
+           Kevin Gorham
+           Jack Grigg
+           Daira Hopwood
+           Linda Naeun Lee
+           George Tankersley
+           Henry de Valence
+  Status: Draft
+  Category: Standards Track
+  Created: 2019-07-17
+  License: MIT
+
+Terminology
+===========
+
+The key words "MUST", "MUST NOT", "SHOULD", and "MAY" in this document are to be 
+interpreted as described in RFC 2119. [#RFC2119]_
+
+Zcash protocol terms are as defined in the Zcash Protocol Specification. [#protocol]_
+
+.. _applink:
+By “applink” we mean a platform mechanism for triggering local applications using HTTP links, such as `App Links`_ on Android, `Universal Links`_ on iOS, or `App URI handlers`_ on Windows.
+
+
+Abstract
+========
+
+This proposal defines a mechanism for sending a Zcash payment encapsulated in a URI string. This enables sending Zcash funds over any secure channel, such as via a messaging app, even if the recipient does not yet have Zcash software installed and does not have a Zcash payment address. This is implemented by having the URI convey the secret spending key of an ephemeral Zcash “wallet address”, to which the funds have been transferred. Anyone who learns the URI can accept this payment, by a “finalization” process which uses the key given in the URI to transfer the encapsulated funds to their own wallet. After the payment is finalized, via a suitable on-chain transaction by the recipient, it becomes irrevocable.
+
+The proposal specifies the syntax and semantics of Payment-Encapsulating URIs, the workflow of generating and handling them, and requisite infrastructure.
```

Comment:
Is it URI-Encapsulated Payments or Payment-Encapsulating URIs

Reply:
Nice catch. Addressed in f20bdfcb793435229118e7c4824d7f6d69f3b27d

Reply:
[Note] mirrored over to the google doc draft:
(this section had been completely removed and this edit no longer applies)